demos/induction_heads_multigpu.py

```diff
@@ -283,8 +287,8 @@ def main(args: Namespace) -> None:
         "batch seq_len -> batch (2 seq_len)",
     )
 
-    model, config = setup_model(args.model_path)
-    fsdp_cfg = fsdp_config()
+    model, config = setup_model(args.model_path, local_rank)
+    fsdp_cfg = fsdp_config(local_rank)
```

Comment:
Use global ranks (ie. `torch.distributed.get_rank(...)`) instead of `local_rank` for these setup methods, since users may be using this demo across node boundaries.